deploy/olm-catalog/qdrouterd-operator/0.1.0/catalog-source.yaml

```diff
@@ -0,0 +1,331 @@
+                              type: string
+                              description: Host name
+                            port:
+                              type: integer
+                              description: Port number
+                            routeContainer:
+                              type: boolean
+                              description: Indicator for a router-broker connection
+                            http:
+                              type: boolean
+                              description: Accept HTTP connections
+                            cost:
+                              type: integer
+                              description: Cost metric for inter router connections
+                            sslProfile:
+                              type: string
+                              description: Name of the ssl profile to use
+                      addresses:
+                        type: array
+                        description: Configuration of each address distribution and phasing
+                        items:
+                          type: object
+                          description: address configuration
+                          properties:
+                            prefix:
+                              type: string
+                              description: The address prefix for the configured setting
+                            pattern:
+                              type: string
+                              description: A wildcarded pattern for address matching
+                            distribution:
+                              type: string
+                              description: Treatment of traffic associated with the address
+                              enum:
+                                - balanced
+                                - closest
+                                - multicast
+                            waypoint:
+                              type: boolean
+                              description: Indicator for waypoint use
+                            ingressPhase:
+                              type: integer
+                              minimum: 0
+                              maximum: 9
+                              description: Ingress phase override for the address
+                            egressPhase:
+                              type: integer
+                              minimum: 0
+                              maximum: 9
+                              description: Egress phase override for the address
+                            priority:
+                              type: integer
+                              minimum: 0
+                              maximum: 9
+                              description: Priority assigned to address for inter router transfer
+      packages: >
+        - #! package-manifest: deploy/catalog_resources/redhat/businessautomation-operator.v1.0.0.clusterserviceversion.yaml
```

Comment:
Should this be updated e.g. replace businessautomation-operator?

Reply:
Yes, and I've update and pushed.